tools/protoxform_test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+rm -rf bazel-bin/tools
+
+declare -r PROTO_TARGETS=$(bazel query "labels(srcs, labels(deps, //tools/testdata/protoxform:protos))")
+
+BAZEL_BUILD_OPTIONS+=" --remote_download_outputs=all --strategy=protoxform=sandboxed,local"
+
+bazel build ${BAZEL_BUILD_OPTIONS} //tools/testdata/protoxform:protos --aspects \
```

Comment:
I have some refactoring to make this nicer in https://github.com/envoyproxy/envoy/pull/9258